src/components/shared/TreeView.js

```diff
@@ -122,15 +125,18 @@ class TreeViewRowFixedColumns<DisplayData: Object> extends React.PureComponent<
       columns,
       index,
       selected,
+      rightClicked,
       highlightRegExp,
       rowHeightStyle,
     } = this.props;
-    const evenOddClassName = index % 2 === 0 ? 'even' : 'odd';
     return (
       <div
-        className={`treeViewRow treeViewRowFixedColumns ${evenOddClassName} ${
-          selected ? 'selected' : ''
-        }`}
+        className={classNames('treeViewRow', 'treeViewRowFixedColumns', {
+          even: index % 2 === 0,
+          odd: index % 2 !== 0,
+          selected,
+          rightClicked,
+        })}
```

Comment:
Note: the change to using `classNames` is what gives the snapshot changes in this commit.